Bottom: 666d20a3198dccc71b64014968f926e8746268f6
Top:    9eec67024c3c2c75c707812d1eb8fb3c06ceff73
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-18 14:09:28 -0500

Refresh of common-install-latest-available

---
diff --git a/roles/docker-buildsystem/tasks/main.yml b/roles/docker-buildsystem/tasks/main.yml
index 382a979..23d9fae 100644
--- a/roles/docker-buildsystem/tasks/main.yml
+++ b/roles/docker-buildsystem/tasks/main.yml
@@ -9,19 +9,6 @@
     - buildsystem_database_root_password is defined
     - docker_restore_config_base_dir is defined
 
-
-- name: Check that docker is installed
-  yum:
-    name: docker
-    state: present
-
-
-- name: Make sure docker service is started
-  service:
-    name: docker
-    state: started
-
-
 - name: Make directory for checkout
   file:
     mode: 0775
